src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
 public @interface Component {
```

Comment:
According to the checklist, don't forget to add `@Target` to custom annotations. This helps to specify where your annotation can be used, for example, on classes, methods, fields, etc. You can add it like `@Target(ElementType.TYPE)` to specify that `@Component` can only be used on types (classes, interfaces, etc.).